knowledge-base/how-to-fix-axe-errors-for-html5-viewer.md

```diff
@@ -0,0 +1,110 @@
+---
+title: How to fix Axe accessibility violations of HTML5 report viewer
+description: How to fix errors reported by Axe on the page that contains HTML5 report viewer
+type: how-to
+page_title: Fix Axe Accessibility Violations of HTML5 Report Viewer
+slug: how-to-fix-axe-errors-for-html5-viewer
+position: 
+tags: accessibility,html5reportviewer
+ticketid: 1451834
+res_type: kb
+---
+
+## Environment
+<table>
+    <tbody>
+	    <tr>
+	    	<td>Product Version</td>
+	    	<td>11.2.17.913 or higher</td>
+	    </tr>
+	    <tr>
+	    	<td>Product</td>
+	    	<td>Progress® Telerik® Reporting</td>
+	    </tr>
+    </tbody>
+</table>
+
+
+## Description
+[Axe web accessibility testing tool](https://www.deque.com/axe/) might report violations and warnings releated to HTML5 Report Viewer and the content of the report.
+
+When providing accessibility for the report viewer we were guided by [Section 508 of the Rehabilitation Act](http://www.section508.gov/), 
+[Web Content Accessibility Guidelines (WCAG) 2.0](https://www.w3.org/TR/WCAG20/), and the standards set by [WAI-ARIA](https://www.w3.org/TR/wai-aria-practices/). 
+The implemented approach was tested using [Microsoft Narrator](https://en.wikipedia.org/wiki/Microsoft_Narrator) and [JAWS](https://www.freedomscientific.com/products/software/jaws/)
+screen readers.
+
+As there is no common stardard for implementing the accessibility for web applications some of the accessibility testing tools might 
```

Comment:
actually the common standards are coverec by WCAG. I would say they are not specific or precise or detailed... something like this.